payment-request/payment-request-constructor.https.html

```diff
@@ -30,6 +30,13 @@
   const request1 = new PaymentRequest(defaultMethods, defaultDetails);
   const request2 = new PaymentRequest(defaultMethods, defaultDetails);
   assert_not_equals(request1.id, request2.id, "UA generated ID must be unique");
```

Comment:
It's a "fail fast". Technically, you are right that we probably don't need it ... but it's pretty cheap and gets the point across so I'm just going to leave it.